CONTRIBUTING.md

```diff
@@ -1,139 +1,138 @@
 
-```sh
-npm install
-```
+If something isn’t working the way it's supposed to, here’s how you can let us know: 
 
-**ProTip**: You can also use [Yarn], which tends to install dependencies more quickly than npm.
+#### 1. Check the issues backlog to see if your bug has already been reported
 
-To start the [StorybookJS] server, run:
+First, check our [current issues backlog](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22+) to see if your bug has already been reported. 
 
-```sh
-npm start
-```
+If your bug has already been reported, leave a comment in the original issue and provide any additional context (if different than the original submission). This helps us better understand the issue and its impact.
 
-Then, visit [http://localhost:6006/](http://localhost:6006/) in a web browser to peruse the component library. While the server is running, any changes that you make to the component templates or configurations will reload the page automatically.
+#### 2. Document how to reproduce the bug
 
-If you're working on the JavaScript or CSS, you can run the "watch" task in
-another shell to automatically rebuild the distribution files that StorybookJS
-references with:
+Before submitting a bug, try to recreate it and document the steps we can take to reproduce it. If you can, take screen shots to capture specific details about the bug. This helps us understand its context. We can only fix bugs that we're able to understand and reproduce.
 
-```sh
-npm run watch
-```
+#### 3. Submit an issue
 
-### Testing
+If your bug or issue is not in the [current issues backlog](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22+), submit an issue using the [bug report template](https://github.com/uswds/uswds/issues/new?assignees=&labels=type%3A+bug&template=bug_report.md&title=). A USWDS core team member may reach out to you if we need further clarification or context. We may also need your help testing possible solutions. Be sure to check in on your issue to answer any questions we may have about it.  
 
-To run the component unit tests, run:
+If you have a code fix for the issue, go ahead and submit a pull request. Our team tracks issues, so make sure any pull request you submit has a related issue.
 
-```sh
-npm test
-```
 
-This will also run [eslint] and [stylelint] to ensure that the JavaScript
-and SCSS source files meet our coding standards along with [snyk test] to check for package dependency vulnerabilities. To lint without the unit
-tests, you'll need [Gulp][]. Install it globally (`npm install -g
-gulp-cli`), then run:
+### 💡 Proposing feature requests or enhancements
 
-```sh
-npm run lint // JS linting
-gulp lintSass
-```
+If you’ve got an idea on how something could work better (enhancement) or on new functionality (feature request) for the Design System, we want to hear about it. Here’s a few steps to help you submit an enhancement or feature request. 
```

Comment:
@elisaachen @bonnieAcameron 
I wonder if we are confusing users with the distinction between "enhancement" and "feature request" here. 

Will users get confused if they want to submit an enhancement but only see instructions for submitting a feature request? When they search the feature request backlog, will that also show issues that are enhancements? 

For the purposes of this guide, would it possibly be clearer if we leave out the distinction between the two? Maybe the distinction is more for our benefit than the users?

Reply:
I really appreciate and agree with this suggestion @amyleadem! I think that 1) removing the definitions in the parentheses will help open it up a bit more and make people feel like they don't have to know exactly what it is they're contributing to wind up in the right place. We also need to  2) change the wording to "Here ARE a few steps to help you submit..."

Reply:
Agree. Made those deletions in the Google doc for both the website and contributing MD @bonnieAcameron @amyleadem

Reply:
Here's what I've suggested:

>If you’ve got a new idea or a suggestion for how something could work better, we want to hear about it. Here are a few steps to help you submit a feature request or enhancement.